### ISSUE TITLE
프로젝트 완료. 

### ISSUE THREAD

**Issue by JangChulwoon** [JangChulwoon/KaraokeParsing#18]
더 이상 할 수 있는, 정확히는 해보고 싶은 작업이 없어서 프로젝트를 멈춥니다.

(다른 오픈소스를 찾아 떠납니다 ... )

**Comment by qkrwlsdyd95** [comment]
안녕하세요 혹시 TJMEDIA 리모콘 기능을 어떻게 구현하는지 아이디어가 있을지요?

**Comment by jangchulwoon** [comment]
@qkrwlsdyd95  안녕하세요 :)
리모콘 기능이면, 검색 기능 말씀하시는걸까요 ?

**Comment by qkrwlsdyd95** [comment]
네 검색 기능이랑 취소, 시작, 예약 등 노래방 리모콘에 있는 기본 기능들을 구현하고 싶은데 잘모르겠어서, 혹시 아시나싶어서 여쭤봅니다 ㅎㅎ

**Comment by jangchulwoon** [comment]
요 프로젝트는 노래방 번호를 검색하는 프로젝트인데요 :) 
노래방 번호 사이트에서 html 을 파싱해와서 검색하고 있습니다.

말씀하시는 부분이, 검색 후 노래방 기계로 예약 혹은 시작을 원하시는거라면 해당 노래방 업체의 interface 를 문의해주셔야할거 같습니다 :)
